### ISSUE TITLE
Image management list shows date deployed not date captured

### ISSUE THREAD

**Issue by davemc50** [FOGProject/fogproject#154]
Column headers in Image Management - List All Images are

Image Name Image Size Format Captured

The value showing in the Captured column comes from 

images.imageLastDeploy

when it should come from

images.imageDateTime

**Comment by mastacontrola** [comment]
I'm not understanding.

The imageLastDeploy column is only updated when the capture completes. The host's deployed column is updated when a host deploy completes.

**Comment by mastacontrola** [comment]
The imageDateTime field is the date the definition was created, not the date the image was uploaded/deployed or otherwise.

**Comment by davemc50** [comment]
Ok. So the "Image List" management display screen shows a column, with the column header **Captured**

In plain english, I understand that to mean the date the image was captured from the source host.

This date was showing blank, even though the image content was in the list.

When I looked in the raw database, images.imageLastDeploy was 0000-00-00.  That appears correct, I have never deployed that image.

When I updated that field images.imageLastDeploy, for testing purposes, the **Captured** value display on the list image management screen display that value.

It is fine for imageDateTime to be date of creation, although it is of little value, but where is a date for imageLastCaptured stored?

The list screen appears to be incorrect or misleading.  Captured should means what it says - date image sourced - this is an important piece of information, as it allows checking that updates or changes made in the golden image up to a certain date are included.

**Comment by mastacontrola** [comment]
The last captured field is relatively new.  1.2.0 did not have this column if I remember correctly.

I'm going on a limb and guessing you upgraded from a prior version, which is when you noticed the captured field.  If the images have not be captured since the update, No data is the correct answer, it simply has no data.

**Comment by davemc50** [comment]
Clean install from November 2016.  New user.

Image created and captured exactly once, since new install.

Why, if the image has been captured once (or any more than nonce), should the captured column be blank?

Am I misintepreting captured, or is not working?

**Comment by mastacontrola** [comment]
When was the image captured?  When you installed, what version did you install?  What version are you running now?

If you're running a version from November 2016, then it's possible at that point in time the field wasn't being updated.  I assure you it does update now.

**Comment by davemc50** [comment]
Ahh ok.

Image captured 2016-10-20

Running Version 1.3.0-RC-13  SVN Revision: 5962, as clean installed, not updated.

>  I assure you it does update now.
Great, problem solved.

Just an out of date version on my end then, fine I'll just update it.

Thanks for your perserverance.
